pvr.vuplus/addon.xml.in

```diff
@@ -71,7 +71,10 @@
     <description lang="de_DE">VU+ -Oberfläche; Unterstützt Live TV &amp; Aufnahmen, EPG und Timer.</description>
     <description lang="el_GR">Frontend για το VU+. Υποστηρίζει ροές Live TV &amp; Εγγραφές, EPG, Χρονοδιακόπτες.</description>
     <description lang="en_AU">VU+ frontend; supporting streaming of Live TV &amp; Recordings, EPG, Timers.</description>
-    <description lang="en_GB">Enigma2 frontend; supporting streaming of Live TV &amp; Recordings, EPG, Timers, Autotimers.</description>
+    <description lang="en_GB">Enigma2 frontend - supporting streaming of Live TV &amp; Recordings, EPG, Timers, Autotimers.
+    
+For documentation visit: https://github.com/kodi-pvr/pvr.vuplus/blob/master/README.md
```

Comment:
This looks strange. Is the "hard" formatting intended?

Reply:
Yes it is. To get the text to appear at the left the line needs to start as above.